fxa/__init__.py

```diff
@@ -6,9 +6,8 @@
 Python library for interacting with the Firefox Accounts ecosystem.
 
 """
-import pkg_resources
 
-__version__ = pkg_resources.get_distribution("PyFxA").version
+__version__ = '0.0.9.dev0',
```

Comment:
since this module has no code, you can actually import __version__ in the package setup.py and avoid duplication